source/declarations.tex

```diff
@@ -34,6 +40,13 @@
     module-import-declaration
 \end{bnf}
 
+\begin{bnf}
+\nontermdef{special-declaration}\br
+    explicit-instantiation\br
+    explicit-specialization\br
+    export-declaration
```

Comment:
_name-declaration_ is missing having these removed.
![1668605278](https://user-images.githubusercontent.com/21071787/202193205-3143aec7-eaed-45fe-85b1-f1824b608bc9.png)

Reply:
Thank you, well spotted! Fixed.